src/chrome/PageControl.tsx

```diff
@@ -59,7 +59,7 @@ const PageControl: React.FC<PageControlProps> = ({
             pageCount={pageCount}
             pageRangeDisplayed={5}
             marginPagesDisplayed={2}
-            pageClassName='bg-gray-200 px-2.5 py-2 font-semibold transition rounded-sm m-1 leading-3 hover:bg-white hover:text-dangerred'
+            pageClassName='pagination_controls bg-gray-200 px-2.5 py-2 font-semibold transition rounded-sm m-1 leading-3 hover:bg-white hover:text-dangerred'
```

Comment:
I don't think this is the right approach.  To vertically center the text, just make sure the height of the container is equal to the line height of the text.  (add an inline `style` attribute setting the `lineheight` and `height` to the same value

Reply:
The ReactPaginate component doesn't accept inline styles. This was my first approach. @chriswhong